src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -421,8 +424,9 @@ public boolean restartJenkins() throws Exception {
      * @return
      * @throws Exception
      */
-    private boolean executeJobTriggering(String jobName, String jobToken, String buildType, String mediatype,
-                                         Map<String, String> parameters, String body)
+    private boolean executeJobTriggering(String jobName, String jobToken, String buildType,
```

Comment:
We don't seem to validate what the jobName is, and then we concatenate a string based of it. What if a user sends in an extra / with the jobname? Are there any characters which are not allowed? The allowed characters should be stated in javadoc for the public methods, and we should add validation for the parameter (+ test).
Should a separate issue be made for this perhaps, since it's out of scope for this PR?

Reply:
This one is inconsitent. In some cases null and empty is checked. I think I corrected one of those cases where null was checked but not empty. I could add some more checks or make an issue.

Reply:
I created an issue here https://github.com/eiffel-community/eiffel-commons/issues/45

Reply:
I looked through the file and thought it was unnecessary to have an issue on this. I'll simply add a check instead and close the issue.